config/seata-config-zk/src/main/java/io/seata/config/zk/ZookeeperConfiguration.java

```diff
@@ -245,4 +250,23 @@ public void handleDataDeleted(String s) {
         }
     }
 
+    private ZkSerializer getZkSerializer() {
+        ZkSerializer zkSerializer = null;
+        String serializer = FILE_CONFIG.getConfig(FILE_CONFIG_KEY_PREFIX + SERIALIZER_KEY);
+        if (StringUtils.isNotBlank(serializer)) {
+            try {
+                Class<?> aClass = Class.forName(serializer);
+                zkSerializer = (ZkSerializer) aClass.newInstance();
+            } catch (ClassNotFoundException cfe) {
+                LOGGER.error("No zk serializer class found, serializer:{}", serializer, cfe);
+            } catch (Exception e) {
```

Comment:
Exception replace to Throwable ? Class.forName can be appears LinkageError

Reply:
Done.